network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -309,6 +309,7 @@ def categories_page(self, request, slug):
 
         context['category'] = slug
         context['current_category'] = category
+        context['pagetype'] = 'category_page'
```

Comment:
@fessehaye Do we still need this information after the latest changes in the template?

Reply:
removed